Sources/KlaviyoSwift/KlaviyoModels.swift

```diff
@@ -87,6 +88,19 @@ public struct Event: Equatable {
         self.uniqueId = uniqueId ?? environment.analytics.uuid().uuidString
         self.identifiers = identifiers
     }
+
+    public init(name: EventName,
+                properties: [String: Any]? = nil,
+                value: Double? = nil,
+                uniqueId: String? = nil) {
+        metric = .init(name: name)
+        _profile = AnyCodable([:])
+        _properties = AnyCodable(properties ?? [:])
+        identifiers = nil
+        self.value = value
+        time = environment.analytics.date()
```

Comment:
shouldn't this be `self.identifiers` and `self.time`?

Reply:
Not sure I'm totally following but you don't have to use self if it's clear what you are setting. I think I have a swiftlint rule that will remove this but I can try setting it.